recipes/linearpartition/meta.yaml

```diff
@@ -1,17 +1,20 @@
 {% set name = "LinearPartition" %}
-{% set version = "1.0" %}
-{% set sha256 = "4fdea96f7ffbd4804d9308ddb46db5f96d1abc4b7bd737725f9bedcae3c88178" %}
+{% set version = "1.0.1.dev20240123" %}
```

Comment:
We usually don't include dev versions. Can you wait for a proper release?

Reply:
`linearpartition` is a C++ package, but there is a small python2 frontend script that calls the C++ executable (which only accepts 10+ undocumented positional arguments).

As far as I understand, one cannot have python2 and python3 installed in an environment at the same time, which ergo doesn't allow the existing linearpartition package to be used in an env when using other python3 scripts/libs.

It is not clear when and if upstream will tag another release, see the linked upstream issue at https://github.com/bioconda/bioconda-recipes/pull/45470. There was an upstream issue  open for a year requesting a new release for `linearfold`, with no upstream reaction.

And I can understand that upstream might not consider the problem to be urgent, because one only interfaces with the python2 driver script from the command-line, and it is not their fault that conda cannot handle installing python2 and python3 simultaneously (correct me if I am wrong).

Reply:
I see thanks for explaining.